index.html

```diff
@@ -5,6 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Trent Sell Audio Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```

Comment:
The font wasn't working because you weren't including it in the project.
I went here and copy pasted import code and it seemed to work: https://fonts.google.com/selection/embed
You can also check this out for an explanation of how web fonts work: https://fonts.google.com/knowledge/using_type

Reply:
What is the best practice with fonts... using an import code or storing the font files locally on your computer? Or does it not matter?

Reply:
I'm not really sure what the current best practice is. I would say just do whatever works for now